app/views/shared/_navbar.html.slim

```diff
@@ -9,11 +9,22 @@ nav.navbar.navbar-expand-lg.navbar-dark.bg-dark
       
         li= link_to 'About',about_path, class:"nav-link"
       
-        li= link_to 'Add Car', new_car_path , class:"nav-link"
+        li= link_to 'Add Car', new_car_path, class:"nav-link"  
 
+        - if @cars.present?
```

Comment:
this should be in helper to fetch all users' cars
also, what if the user won't be logged in?

Reply:
It is nice and east with helper I implement it with big pleasure , it done now